Source/JavaScriptCore/interpreter/CallFrameInlines.h

```diff
@@ -96,6 +96,13 @@ inline bool CallFrame::isStackOverflowFrame() const
     return jsCallee() == jsCallee()->globalObject()->stackOverflowFrameCallee();
 }
 
+inline bool CallFrame::isHandleHostCallExceptionFrame() const
```

Comment:
nit: based on this patch, I don't see a need to distinguish `handleHostCallExceptionCallee` from `stackOverflowFrameCallee`.  We could just rename `stackOverflowFrameCallee` to `partiallyInitializedFrameCallee` and just use that instead.  This way, we don't have to spend time, code, and memory instantiating a second instance of this callee per global.

Reply:
That sounds good.